.github/workflows/LocalesTest.yml

```diff
@@ -0,0 +1,65 @@
+name: Test Different Locales
+
+on:
+  push:
+    branches: [ v9.0 ]
+  pull_request:
+    branches: [ v9.0 ]
+  schedule:
+    - cron: 0 6 * * 1
+  
+env:
+  LOCALE_LANG: "fr_FR.UTF-8"
+
+jobs:
+  test-french-locale:
+    name: Test French Locale On Ubuntu
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Any reason this is set to false?

Reply:
I think as I am using a matrix currently, I am going to keep it for now while finding the bugs, but we could remove it later